crates/turbopack-core/src/ident.rs

```diff
@@ -61,8 +61,18 @@ impl ValueToString for AssetIdent {
             write!(s, "#{}", fragment.await?)?;
         }
 
-        for (key, asset) in &self.assets {
-            write!(s, "/({})/{}", key.await?, asset.to_string().await?)?;
```

Comment:
I found this super confusing whenever I was looking at it

Reply:
Sounds good, but could you move that into a separate PR as it affects all snapshots